ixwebsocket/IXWebSocket.cpp

```diff
@@ -142,6 +142,7 @@ namespace ix
     {
         bool automaticReconnection = _automaticReconnection;
 
+
```

Comment:
nitpick: I like tidy code, I don't think we need that extra \n

Reply:
Added with std::cout to debug, my bad